.kokoro/autorelease.sh

```diff
@@ -15,12 +15,7 @@ populate_all_secrets
 python -m pip install --require-hashes -r pip-requirements.txt
 python -m pip install --require-hashes -r requirements.txt
 
-# The publish reporter script uses "python3" which doesn't exist on Windows.
-# Work out what we should use instead, and fix up the script.
-python -m releasetool publish-reporter-script > /tmp/publisher-script
-PYTHON3=$(source ../toolversions.sh && echo $PYTHON3)
-sed -i "s/python3/$PYTHON3/g" /tmp/publisher-script
-source /tmp/publisher-script
+../tools/Google.Cloud.Tools.ReleaseTool/publish_reporter.sh
```

Comment:
I suspect this should use "source" - otherwise it'll start a new shell and report when *that* completes.

Reply:
Done